packages/glsp-sprotty/src/index.ts

```diff
@@ -73,7 +73,9 @@ export {
     SModelElementConstructor as GModelElementConstructor,
     SModelElementRegistration as GModelElementRegistration,
     IModelFactory,
+    // exported without alias we extend it in glsp-client to `SModelFactory`
```

Comment:
Could you show me where that is happening? For some reason I cannot find it. It seems we have our own definition of `GModelFactory` and `SModelFactory` is not really used or am I just too blind to find it?

Reply:
Sorry my bad. You are right we only extend the `SModelRegistry` on the client. Confusion from my side, we can simply alias `SModelFactory`